src/pyOpenMS/R-addons/DPosition.R

```diff
@@ -0,0 +1,78 @@
+
```

Comment:
You provide some R-style documentation here but not in general. The standard roxygen style documentation is like this (from https://r-pkgs.org/man.html):
#' Add together two numbers
#' 
#' @param x A number.
#' @param y A number.
#' @return The sum of \code{x} and \code{y}.
#' @examples
#' add(1, 1)
#' add(10, 1)
add <- function(x, y) {
  x + y
}